aviary/models/large_single_aisle_1/large_single_aisle_1_GwGm.csv

```diff
@@ -56,7 +56,8 @@ aircraft:fuselage:pressure_differential,7.5,psi
 aircraft:fuselage:seat_pitch,29,inch
 aircraft:fuselage:seat_width,20.2,inch
 aircraft:fuselage:tail_fineness,3,unitless
-aircraft:fuselage:wetted_area_factor,4000,unitless
+aircraft:fuselage:wetted_area,4000,ft**2
+aircraft:fuselage:wetted_area_scaler,1,unitless
```

Comment:
Just a note, that you don't need to (and probably shouldn't) set the area scaler when also specifying the area, because the scaler is essentially ignored.

Reply:
It is how it is done on the FLOPS side: See for example: `models/test_aircraft/aircraft_for_bench_FwFm.csv`. Can you double check?